images/toolbox/toolbox.sh

```diff
@@ -47,6 +47,7 @@ EOF
     MON_ENDPOINTS=$(echo $CLIENT_INFO | jq -r '.monAddresses[]' | awk -F '/' 'BEGIN { ORS="," }; {print $1}' | sed 's/,$//')
     KEYRING_FILE="/etc/ceph/keyring"
 
+    mkdir -p /etc/ceph
```

Comment:
the purpose for this toolbox entrypoint script is to discover information from the environment and then create a minimal ceph config file in the default location (`/etc/ceph/ceph.config`) so that the ceph tools can be used by a human with 0 additional arguments (e.g. `ceph status` just magically works).  That makes sense for the toolbox, but I'm skeptical of it for rookd since that config file in the default location could potentially confuse all the ceph tools that rookd shells out to now.

Therefore, I could create `/etc/ceph` in the dockerfile for the toolbox (instead of this entrypoint script), but I don't think I should touch the rookd dockerfile.